src/pages/docs/creators/ray/index.md

```diff
@@ -6,24 +6,79 @@ description: Explore how to use Ray with Golem for scaleable Python code executi
-- [Concept](/docs/creators/ray/concept) - guide on what Ray and Ray on Golem is, listing all pieces needed to use them
 - [Setup tutorial](/docs/creators/ray/setup-tutorial) - step-by-step explanation of the quickstart instructions
+- [Supported versions and other limitations](/docs/creators/ray/supported-versions-and-other-limitations) - list of supported python and ray versions, and other known issues
 - [Basic Ray tasks usage tutorial](/docs/creators/ray/basic-ray-tasks-usage-tutorial) - anatomy of Ray application based on a simple code showcasing Ray tasks
 - [Converting a real-life use case to Ray on Golem](/docs/creators/ray/conversion-to-ray-on-golem-tutorial) - example of a little bit more practical Ray on Golem usage
-- [Cluster yaml reference](/docs/creators/ray/cluster-yaml-reference) - details of every configuration option of Ray on Golem cluster yaml
-- [Supported versions & other limitations](/docs/creators/ray/supported-versions-and-other-limitations) - list of supported python & ray versions, and other known issues
+- [Cluster yaml](/docs/creators/ray/cluster-yaml) - details of configuration options of Ray on Golem cluster yaml
+- [Ray CLI](/docs/creators/ray/ray-cli) - details of Ray command line options used with Ray on Golem
 - [Troubleshooting](/docs/creators/ray/troubleshooting) - hopefully helpful tips on what to do when something goes wrong
 
 
+## Introduction
+
+Golem is a decentralized marketplace for computing power, where the providers let the requestors use their machines for a small fee.
+
+Ray on Golem makes it super easy to set up and use Golem Network to run your Ray application.
+
+Ray on the other hand is an open-source framework for scaling Python applications. 
+It specializes in tooling for AI/ML applications, but at the same time, it is based on Ray Core which understands every piece of generic Python code.
+
+
+## Why Ray?
+
+The best thing about Ray is that it makes it super easy to parallelize existing Python code even if you are not a distributed programming expert.  
+
+Ray uses concepts of tasks, actors, and objects for building and scaling distributed applications.
+It can be used to parallelize your Python code to use all cores on your computer, but more importantly, it also offers a Ray Cluster interface to run your payload on several, remote machines.
+
+The beauty of the solution is that you can run the app locally, and when you are ready, you can execute the same app on the cluster without changing a single line of your code.
+
 {% partial file="ray/learn-more-about-ray.md" /%}
 
-If you have any questions, comments insights, praises, or doubts about these docs and Ray on Golem in general please don't hesitate to reach out to us either on
+## What do you need
+
+To use Ray on Golem, you need to install the Ray framework and Ray on Golem cluster manager. 
```

Comment:
this suggests that the operation is more involved that it really is ... ... 

I somehow feel this sentence doesn't carry any useful meaning... 

```suggestion
You only need to install Ray on Golem - by default it also installs the Golem daemon and the Ray framework as dependencies.
```

Reply:
My intended meaning is to warn people that they will have yagna and ray stuff running on their computers :)